tools/compare_runs.py

```
@@ -0,0 +1,250 @@
+    pending_refs = args.ref if len(targets) < 2 else []
+    if len(targets) + len(pending_refs) != 2:
+        sys.exit("需要正好两个比较对象：给两个 --ref，或两个 --binary")
+
+    out_dir = Path(args.out) if args.out else Path(tempfile.mkdtemp(prefix="spl_compare_"))
+    out_dir.mkdir(parents=True, exist_ok=True)
+    work = out_dir / "work"
+    work.mkdir(exist_ok=True)
+
+    created_worktrees = []
+    try:
+        for offset, ref in enumerate(pending_refs):
```

Ruff (0.16.1)

[warning] Loop control variable `offset` not used within loop body

Rename unused `offset` to `_offset`

(B007)